numba/ir_utils.py

```diff
@@ -11,7 +11,8 @@
 
 import numba
 from numba.six import exec_
-from numba import ir, types, typing, config, analysis, utils, cgutils, rewrites
+from numba import (ir, types, typing, config, analysis, utils, cgutils,
+    rewrites, postproc)
```

Comment:
postproc indirectly imports ir_utils, so this won't work on Python 2.7.  We need to reshuffle things to break the circularity.